src/contentScripts/storage.ts

```diff
@@ -6,10 +6,12 @@ export const repos = useStorage<{
   lastUpdated: number
   recent: string[]
   pinned: string[]
+  details: Record<string, string>
```

Comment:
```suggestion
  details: Record<string, { alias?: string }>
```

Let's have this for future extensibility.

Reply:
OK

Reply:
@antfu my fix is ready, please help review the code.